src/screens/Portfolio/index.tsx

```diff
@@ -33,11 +33,26 @@ const renderScene = ({ route }: RenderSceneProps) => {
   }
 };
 
-export const PortfolioScreen = () => {
-  const [index, setIndex] = useState(0);
+type PortfolioScreenProps = {
+  route: {
+    params: {
+      tabs: { activeTab: number };
+    };
+  };
+};
+
+export const PortfolioScreen = ({ route }: PortfolioScreenProps) => {
+  const { activeTab } = route.params.tabs;
+
+  const [index, setIndex] = useState(activeTab);
   const [routes] = useState<PortfolioTabViewRoute[]>(
     portfolioTabRoutes as unknown as PortfolioTabViewRoute[]
   );
+
+  useLayoutEffect(() => {
```

Comment:
I think it will not work if you navigate into PortfolioScreen multiple times. You can use, `useIsFocused` hook from `react-navigation` and add `focused` as a dependency of `useLayoutEffect`. Then check if it is focused inside and if there in `activeTab` param.